### ISSUE TITLE
Few comments on wording and objectives

### ISSUE THREAD

**Issue by sergueev** [swcarpentry/git-novice#316]
Hello,

After proofreading the Version Control with Git, I happened to have few comments. 

- First, in the Conflicts episode there is a sentence at the end:  "Version control’s ability to merge conflicting changes is another reason users tend to divide their programs and papers into multiple files instead of storing everything in one large file."   I am not sure whether it's only me but I find that the wording of this sentence is a bit strange. First part  "Version control's ability" sounds like an advantage one can profit from. The second part of the sentence sounds like no one wants to profit from this advantage but instead collaborate on different files. 
- Second, in the Collaborating episode there is a discussion of a basic collaborative workflow which I think deserves to be mentioned in the Objectives of the episode (something like "How to collaborate effectively" or "Basic strategies for collaboration with Git").
Thanks.

**Comment by iglpdc** [comment]
The second point is covered by #490.

**Comment by shals3** [comment]
@iglpdc  I could not find the first issue in the Conflicts episode which is why I did not make any changes.

**Comment by iglpdc** [comment]
You're right, @shals3. It must have been changed somewhere else. I'm closing this. Thanks again.
